src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/EnumShouldNotHaveDuplicatedValuesTests.cs

```diff
@@ -246,6 +309,27 @@ public enum MyEnum
 End Enum");
         }
 
+        [Fact]
+        public async Task EnumValueReferenceExplicitDuplicateMember_Diagnostic()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+public enum MyEnum
+{
+    Value1 = 1,
+    Value2 = Value1,
+    Value3 = Value2,
```

Comment:
This is a behavior change. The idea here is that if you are going to explicitly include a duplicate, its initializer should reference the same "original". To fix this code the user should change `Value3 = Value2` to `Value3 = Value1`. I felt like this was in the spirit of the [docs](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1069) but I'm also fine with changing the behavior back so this test produces no diagnostics.

Reply:
I think this makes sense here, but might not for bitwise enums. I'll try to find an example.

Reply:
Consider [MethodAttributes](https://github.com/dotnet/runtime/blob/4c92aef2b08f9c4374c520e7e664a44f1ad8ce56/src/libraries/System.Private.CoreLib/src/System/Reflection/MethodAttributes.cs) for example. The members `PrivateScope` and `ReuseSlot` have the same value; if another member were added to the end with the intended value `Family | ReuseSlot`, it would be annoying to have a warning appearing saying you need to put `Family | PrivateScope` instead (which is technically the same value, but clearly less meaningful).

Reply:
Have added a test for this scenario in 37d8ae9159e0cb1e4b430e259840c2c2c6dfd0e6. It appears the diagnostics given have not changed from before this PR.